python/pyspark/ml/wrapper.py

```diff
@@ -159,15 +151,16 @@ class JavaModel(Model, JavaTransformer):
 
     __metaclass__ = ABCMeta
 
-    def __init__(self, java_model):
+    def __init__(self, java_model=None):
```

Comment:
Unify the construction of ```Model``` and ```Estimator```, that is ```Model``` can be instantiated without argument which is used when ```model.load```.